scripts/linux/buildBranch.sh

```diff
@@ -212,14 +202,7 @@ build_solution()
     echo "Building IoT Edge solution"
     dotnet --version
 
```

Comment:
I think the script should be agnostic. It should just do `dotnet build`, and dotnet will use the framework given in the project file. I think we specified the framework on the command line because the dotnet CLI used to require it in certain circumstances, but the tooling has since matured.

These changes make the script _mostly_ agnostic to the framework version...I thing the old-style test apps (e.g., IotEdgeQuickstart) are still being built with an explicit framework version; I didn't mess with them.